README.md

```diff
@@ -11,6 +11,13 @@ These are regexes that could lead to [catastrophic backtracking](https://www.reg
 2. Run the `configure` script to install dependencies and build the detectors.
 3. Use the scripts in `bin`. See their [README](https://github.com/davisjam/vuln-regex-detector/blob/master/bin/README.md) for details.
 
+Alternately, a `Dockerfile` is provided to allow the project to run on non-Ubuntu systems. The image can be built and used as follows:
+
+```shell
+$ docker build -t vuln-regex-detector .
+$ docker run --rm vuln-regex-detector bin/check-tree.pl tree.json
```

Comment:
1. I believe this example won't work unless there is a `tree.json` file built into the container.

I think incorporating `--mount` seems like it would be helpful in most use cases.
What do you think about offering something like this as the example instead?

```
docker run \
--mount type=bind,source=/tmp/query,target=/query \
--rm \
vuln-regex-detector bin/check-regex.pl /query/check-regex/unsafe-1.json
```

Reply:
Yeah that's true, I was actually running it using `-v` (because it's a bit more concise and I don't have a need for bind mounts), like this:

```
docker run --rm -v /tmp/query:/query vuln-regex-detector bin/check-regex.pl /query/unsafe-1.json
```

Happy to change to whichever you prefer.

Reply:
Yeah that's true, I was actually running it using `-v` (because it's a bit more concise and I don't have a need for bind mounts), like this:

```
docker run --rm -v /tmp/query:/query vuln-regex-detector bin/check-regex.pl /query/unsafe-1.json
```

Happy to change to whichever you prefer.